src/ripple/app/consensus/RCLCxPos.h

```diff
@@ -0,0 +1,142 @@
+//------------------------------------------------------------------------------
+/*
+    This file is part of rippled: https://github.com/ripple/rippled
+    Copyright (c) 2012-2016 Ripple Labs Inc.
+
+    Permission to use, copy, modify, and/or distribute this software for any
+    purpose  with  or without fee is hereby granted, provided that the above
+    copyright notice and this permission notice appear in all copies.
+
+    THE  SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL WARRANTIES
+    WITH  REGARD  TO  THIS  SOFTWARE  INCLUDING  ALL  IMPLIED  WARRANTIES  OF
+    MERCHANTABILITY  AND  FITNESS. IN NO EVENT SHALL THE AUTHOR BE LIABLE FOR
+    ANY  SPECIAL ,  DIRECT, INDIRECT, OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES
+    WHATSOEVER  RESULTING  FROM  LOSS  OF USE, DATA OR PROFITS, WHETHER IN AN
+    ACTION  OF  CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF
+    OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
+*/
+//==============================================================================
+
+#ifndef RIPPLE_APP_CONSENSUS_RCLCXPOSITION_H_INCLUDED
+#define RIPPLE_APP_CONSENSUS_RCLCXPOSITION_H_INCLUDED
+
+#include <ripple/json/json_value.h>
+#include <ripple/basics/chrono.h>
+#include <ripple/protocol/UintTypes.h>
+
+namespace ripple {
+
+// A position taken during a consensus round
+// As seen by the RCL consensus process
+class RCLCxPos
```

Comment:
Its not about saving typing its about identifier fatigue. Repeating the word "Consensus" 117 times in one source file, across six different types, is not self-documenting - its straining.